src/services/onboard.ts

```diff
@@ -41,12 +40,18 @@ export const createOnboard = (chainConfigs: ChainInfo[], rpcConfig: EnvState['rp
       desktop: { enabled: false },
     },
 
+    notify: {
+      enabled: false,
+    },
+
     appMetadata: {
       name: 'Safe{Wallet}',
-      icon: '/images/safe-logo-green.png',
+      icon: '<svg><image href="/images/safe-logo-green.png" width="56" height="56" /></svg>',
```

Comment:
The logo does not render correctly in Chrome anymore unless wrapped in an SVG (as clarified [in the docs](https://onboard.blocknative.com/docs/modules/core#appmetadata)).